app/assets/javascripts/osem-datepickers.js

```diff
@@ -79,6 +79,7 @@ $(function () {
 
   $("#conference-start-datepicker").on("dp.change",function (e) {
       $('#conference-end-datepicker').data("DateTimePicker").setMinDate(e.date);
+      $('#conference-end-datepicker').val()?$('#conference-end-datepicker').val():$('#conference-end-datepicker').data("DateTimePicker").setDate(e.date);
```

Comment:
Although this work, I find this syntax quite difficult to read. And I don't like that you are assigning again the datepicker in case that if was already assigned. What about this:

``` Javascript
unless $('#conference-end-datepicker').val(){
  $('#conference-end-datepicker').data("DateTimePicker").setDate(e.date);
}
```

And maybe we should also assign `$('#conference-end-datepicker')` do a variable as it used several times. 🤔

Reply:
I understand your point of view.

Sorry, but I don't know `unless` (I've tried to find some documentation about, but I didn't find one).

So, an alternative could be:

```javascript
if (!$('#conference-end-datepicker').val()) {
   $('#conference-end-datepicker').data("DateTimePicker").setDate(e.date); 
}
```

Reply:
yes, sorry, `unless` doesn't exist in javascript 🙈 

> So, a better alternative could be:
>
> ```javascript
> if (!$('#conference-end-datepicker').val()) {
>    $('#conference-end-datepicker').data("DateTimePicker").setDate(e.date); 
> }
> ```

This looks great 👍